packages/quantic/force-app/main/default/lwc/quanticRefineModalContent/quanticRefineModalContent.js

```diff
@@ -52,6 +47,13 @@ export default class QuanticRefineModalContent extends LightningElement {
    * @type {boolean}
    */
   @api hideSort;
+  /**
+   * Indicates whether the Dynamic Navigation for facet ordering should be disabled.
+   * @api
+   * @type {boolean}
+   * @defaultValue `false`
+   */
+  @api disableDynamicNavigation = false;
```

Comment:
Since the DNE feature includes more than just the facet ordering like documented here: https://docs.coveo.com/en/3383/leverage-machine-learning/about-dynamic-navigation-experience-dne
What do you think about making the name of this property very specific to what the property does.
`disableFacetOrdering` for example.